src/v2/Artsy/Router/RouterLink.tsx

```diff
@@ -41,6 +41,7 @@ export const RouterLink: React.FC<RouterLinkProps> = ({
 
   if (isSupportedInRouter) {
     const allowedProps = pick(props, [
+      "aria-label",
```

Comment:
Haven't looked into if there's a more comprehensive way to do this yet, but this caught me off guard — having to allowlist props on links. 🤔

Reply:
React will yell if you try to pass non html props down

Reply:
oo interesting I didn't even realize this code block was here!